File: ts/get-localized-content/GetLocalizedItemsWithSlug.ts
// DocSection: getting_localized_content_url_slug
// Tip: Find more about JS/TS SDKs at https://kontent.ai/learn/javascript
import { createDeliveryClient } from '@kontent-ai/delivery-sdk';
import { Article } from './models/Article';

const deliveryClient = createDeliveryClient({
  projectId: '8d20758c-d74c-4f59-ae04-ee928c0816b7',
});

const response = await deliveryClient.items<Article>()
  .type('article')
  .languageParameter('es-ES')
  .depthParameter(0)
  .equalsFilter('elements.url_pattern', 'acerda-de-nosotros')
  .toPromise();
// EndDocSection

File: ruby/get-localized-content/GetLocalizedItemsWithSlug.rb
# DocSection: getting_localized_content_url_slug
# Tip: Find more about Ruby SDKs at https://kontent.ai/learn/ruby
require 'delivery-sdk-ruby'

delivery_client = Kontent::Ai::Delivery::DeliveryClient.new project_id: '8d20758c-d74c-4f59-ae04-ee928c0816b7'
delivery_client.items([
                        'system.type'.eq('article'),
                        'elements.url_pattern'.eq('acerda-de-nosotros')
                      ])
               .language('es-ES')
               .execute do |response|
                 items = response.items
               end
# EndDocSection